jobs/e2e_node/containerd/containerd-master/image-config.yaml

```
@@ -4,6 +4,6 @@ images:
     project: ubuntu-os-gke-cloud
     metadata: "user-data</go/src/github.com/containerd/cri/test/e2e_node/init.yaml,containerd-configure-sh</go/src/github.com/containerd/cri/cluster/gce/configure.sh,containerd-env</workspace/test-infra/jobs/e2e_node/containerd/containerd-master/env"
   cos-stable:
-    image: cos-77-12371-274-0
+    image_family: cos-81-lts
```

As per @bsdnet comment:

```suggestion
    image: cos-81-12871-119-0
```